plugins/dpsnPlugin/.gitignore

```
@@ -0,0 +1 @@
+./coverage
```

```suggestion
./coverage

```
Add a newline as of POSIX standard